clang/lib/Basic/Targets/SystemZ.h

```diff
@@ -115,10 +115,19 @@ class LLVM_LIBRARY_VISIBILITY SystemZTargetInfo : public TargetInfo {
     return RegName == "r15";
   }
 
+  // CC has interval [0, 4).
+  unsigned getFlagOutputCCUpperBound() const override { return 4; }
   bool validateAsmConstraint(const char *&Name,
                              TargetInfo::ConstraintInfo &info) const override;
 
   std::string convertConstraint(const char *&Constraint) const override {
+    if (llvm::StringRef(Constraint).starts_with("@cc")) {
```

Comment:
Why are we trying to support different constraint strings here?  On our platform, it should be enough to check for exact match against `"@cc"`, right?